src/app/(pages)/wdi/_components/ActionLink.tsx

```diff
@@ -0,0 +1,15 @@
+interface Props {
+	href: string;
```

Comment:
We should also move this to src/components and stand arise buttons

Reply:
I assume you've meant `standardize` 😄 Yeah that might be a good idea to have some predefined building blocks